setup.py

```diff
@@ -29,6 +29,7 @@
       maintainer_email="nagappan@gmail.com",
       url="http://ldtp.freesktop.org",
       license="GNU Lesser General Public License (LGPL)",
+      install_requires=["twisted"],
```

Comment:
Why do we need twisted?

Reply:
Otherwise change look good to me

Reply:
Twisted seems to be required

https://github.com/ldtp/ldtp2/blob/master/ldtpd/__init__.py#L24

Reply:
Hi,
    Is this OK to merge ?

S